boards/earlgrey-nexysvideo/src/main.rs

```diff
@@ -6,6 +6,9 @@
 // Disable this attribute when documenting, as a workaround for
 // https://github.com/rust-lang/rust/issues/62184.
 #![cfg_attr(not(doc), no_main)]
+#![feature(custom_test_frameworks)]
```

Comment:
Flagging this nightly feature for others to see -- nightly features in `boards/` are generally less problematic than those in other places, but we should consider that this feature will likely become a requirement for our CI to keep working if we incorporate these tests.

Reply:
I think we the fairly federated CI infrastructure this is okay. We should be able to slice out this test in the future if needed relatively easily.